pkgs/os-specific/linux/lsirec/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lsirec";
+  version = "unstable-2019-03-03";
+
+  src = fetchFromGitHub {
+    owner = "marcan";
+    repo = "lsirec";
+    rev = "2dfb6dc92649feb01a3ddcfd117d4a99098084f2";
+    sha256 = "sha256-8v+KKjAJlJNpUT0poedRTQfPiDiwahrosXD35Bmh3jM=";
+  };
+
+  postPatch = ''
+    substituteInPlace sbrtool.py \
+      --replace '/usr/bin/python3' '${python3}/bin/python'
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/bin"
```

Comment:
Not needed with `install(1)`
> -D     create  all leading components of DEST except the last, or all components of --target-di‐
              rectory, then copy SOURCE to DEST

Reply:
Thanks for your suggestion.